### ISSUE TITLE
UI Polish needed

### ISSUE THREAD

**Issue by philbarker** [t3-innovation-network/desm#579]
We need to go through the workflow and give the UI a polish. 

In addition to several issues already notes (e.g. poor display on mobile devices), in lots of places when we have created new funcitonality we have added elements to the UI in an unplanned manner so that now we a lack of coherence in the interface.

Step one would be to get someone with UI/UX experience who knows what DESM is for to work through the tool and list specifics suggestions for improvement.
